src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/Enrich.scala

```diff
@@ -104,7 +107,7 @@ object Enrich {
     val raw: SCollection[Array[Byte]] =
       sc.withName("raw-from-pubsub").pubsubSubscription[Array[Byte]](config.raw)
 
-    val enriched: SCollection[Validated[List[String], EnrichedEvent]] =
+    val enriched: SCollection[Validated[BadRow, EnrichedEvent]] =
```

Comment:
Can't [1](https://github.com/snowplow/beam-enrich/blob/master/src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/utils.scala#L112) and [2](https://github.com/snowplow/beam-enrich/blob/master/src/main/scala/com.snowplowanalytics.snowplow.enrich.beam/utils.scala#L133) return `BadRow` as well ?

Reply:
Right, thanks for spotting Ben! fixing now